.github/workflows/add-asana-pr-comment.yml

```
@@ -0,0 +1,17 @@
+name: Add PR link to Asana task
+
+on:
+  pull_request:
+    types: [opened, reopened]
```

Tempted to change this to the following, since I can't think of a good reason that we would need to trigger this on `reopened`:

```suggestion
    types: [opened]
```

---

Very good point @samsymons, let's limit it only to opened event. I'll try to commit the suggestion :)